src/lmppython.h

```diff
@@ -20,7 +20,7 @@ namespace LAMMPS_NS {
 
 class PythonInterface {
  public:
-  virtual ~PythonInterface() = default;
+  virtual ~PythonInterface() noexcept(false) {}
```

Comment:
Why can these throw?

Reply:
I only commented on one of these, but the change in Pointers better have a **very** good justification. Otherwise you're just inviting resource leaks and forced termination.